.github/workflows/pr-automation.yml

```
@@ -0,0 +1,186 @@
+          if [ -n "$MISSING_TESTS" ]; then
+            echo "has_missing_tests=true" >> $GITHUB_OUTPUT
+            echo "missing_tests<<EOF" >> $GITHUB_OUTPUT
+            echo -e "$MISSING_TESTS" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          else
+            echo "has_missing_tests=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Post Check Comment
+        if: steps.check-tests.outputs.has_missing_tests == 'true'
+        uses: peter-evans/create-or-update-comment@v3
```

actionlint (1.7.11)

[error] the runner of "peter-evans/create-or-update-comment@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)